tests/cli-paths.rs

```diff
@@ -109,6 +109,7 @@ export PATH="$HOME/apple/bin"
         });
     }
 
+    // This test should NOT be run as root!
```

Comment:
Where did this comment come from?

Reply:
By default, on cirrus CI we run scripts as root. So this test was failing and I couldn't understand why. The reason is simple: when root, the file rights are overlooked, making this test fail. Considering how much time I spent on this issue, I felt like the problem could be solved faster if we add support for another target and encounter the same issue.

Reply:
Ouch. OK